Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/jvm/main.kt

```diff
@@ -282,7 +285,9 @@ private fun processCLib(args: Array<String>, additionalArgs: Map<String, Any> =
         _, oldValue, newValue ->
             warn("The package value `$oldValue` specified in .def file is overridden with explicit $newValue")
     }
-    def.manifestAddendProperties["interop"] = "true"
+    def.manifestAddendProperties[KLIB_PROPERTY_INTEROP] = "true"
+    if (outKtPkg.startsWith("platform."))
+        def.manifestAddendProperties[KLIB_PROPERTY_UNIQUE_NAME] = "$platformPrefixNoPlatform${outKtPkg}"
```

Comment:
May be make it a cli flag?

Reply:
Not sure what is the point in having that outside of platform libraries context.

Reply:
But now this logic is applied to all interop libraries, isn't it? What if a user creates a library with a `project.something` package?